ostree-ext/src/chunking.rs

```diff
@@ -136,35 +136,35 @@ struct Generation {
     dirmeta_found: BTreeSet<RcStr>,
 }
 
-fn push_dirmeta(repo: &ostree::Repo, gen: &mut Generation, checksum: &str) -> Result<()> {
-    if gen.dirtree_found.contains(checksum) {
+fn push_dirmeta(repo: &ostree::Repo, r#gen: &mut Generation, checksum: &str) -> Result<()> {
```

Comment:
This is fine as an automated fix, but I think it'd be nicer to rename this to `generation` instead

Reply:
100%, I did exactly that manually before I realized that `cargo fix` would do its own thing.